### PR TITLE
Update linux instructions

### PR DESCRIPTION
**Pull request by joaonzangoII**
Fixes craftacademy/ca_course#59
- fix formatting
- add instruction to change pretzo theme
- add instructions to install nvn
- add instructions to install node via nvm

### CODE REVIEW

File: terminal/linux.md [hunk 1]
```diff
@@ -233,11 +233,130 @@ There are plenty of frameworks that have been created to help setup zsh configur
...
 
-4. ```
+  ```
   $ prompt <name-of-the-theme>
   ```
 
 
+**Note:** If the above mentioned command does not work, follow the following steps to change the current prompt theme
+
+Go to a terminal windoy and type the following command:
```
Comment (diraulo) [inline terminal/linux.md]: `Go to your terminal ...`

File: terminal/linux.md [hunk 2]
```diff
@@ -233,11 +233,130 @@ There are plenty of frameworks that have been created to help setup zsh configur
...
+Scroll through the file until you find a similar code to the following snippet then replace with the desired theme name as instructed below.
+
+```
+#
+# Prompt
+#
+# Set the prompt theme to load.
+# Setting it to 'random' loads a random theme.
+# Auto set to 'off' on dumb terminals. 
+  zstyle ':prezto:module:prompt' theme '<name-of-theme'>
```
Comment (diraulo) [inline terminal/linux.md]: is this meant to be `'<name-of-theme>'`

**Comment by diraulo** [comment]
@joaonzangoII note how I reference the issue on the other repo in the PR description. 😉

File: terminal/linux.md [hunk 3]
```diff
@@ -233,11 +233,131 @@ There are plenty of frameworks that have been created to help setup zsh configur
...
+  Which in this case would yield a result of:
+
+  ```
+  0.31.3
+  ```
+
+  Another very useful command to get you started on node.js management is:
+
+  ```
+  nvm help
```
Comment (diraulo) [inline terminal/linux.md]: forgot the `$` here.

File: terminal/linux.md [hunk 4]
```diff
@@ -233,11 +233,131 @@ There are plenty of frameworks that have been created to help setup zsh configur
...
   $ prompt <name-of-the-theme>
   ```
 
 
+**Note:** If the above mentioned command does not work, follow the following steps to change the current prompt theme
+
+Go to your terminal window and type the following command:
+
+```
+nano ~/.zpreztorc
```
Comment (diraulo) [inline terminal/linux.md]: forgot to show the prompt on this command. Check it on the other commands in this doc too

**Comment by diraulo** [comment]
@joaonzangoII may you please check the last comments I made and fix those then I'll merge this.

File: terminal/linux.md (hunk 4 above)
Comment (joaonzangoII) [inline terminal/linux.md]: @diraulo what do you mean by `forgot to show the prompt on this command.`?

Comment (diraulo) [inline terminal/linux.md]: the `$` sign in front of shell commands. You have it on some commands and not on others.

**Pull request merged**